icubam/www/handlers/base.py

```diff
@@ -75,6 +76,21 @@ def get_user_locale(self):
       return tornado.locale.get(locale_code)
 
 
+def authenticated(func=None, *, code=503):
```

Comment:
So this just returns an HTTP error code without rendering any template right. Which is what we want for API calls but not user facing endpoints.

Maybe call this `authenticated_api` or put a sentence in the docsting to indicate that it should not be used with non API endpoints?